src/refiners/foundationals/dinov2/vit.py

```diff
@@ -0,0 +1,227 @@
+from typing import Any
+
+import torch
+from torch import Tensor
+
+import refiners.fluxion.layers as fl
+from refiners.fluxion.layers.activations import Activation
+
+
+class CLSToken(fl.Parameter):
+    """Return a learnable class token to the input tensor."""
+
+    def forward(self, x: Tensor) -> Tensor:
+        return self.parameter.expand(x.shape[0], -1, -1)
+
+
+class PosEmbedding(fl.Residual):
```

Comment:
ditto: it is called `PositionalEncoder` in https://github.com/finegrain-ai/refiners/blob/0dc3a17fbf442f5b7f38813d77752dec08bc26db/src/refiners/foundationals/clip/common.py#L5

Let's use this name here too for consistency. Same remark re: implementation which slightly differs - you could make it on par.